### PR TITLE
circleci: initial commit.

### PR DESCRIPTION
**Pull request by StatusReport**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+      - run:
+          name: Build and run tests
+          command: fastlane scan
+          environment:
+            SCAN_DEVICE: iPhone 8
+            SCAN_SCHEME: libExpecta-iOS
+
+      - store_test_results:
+          path: test_output/report.xml
```
Comment (DekelAvrahami) [inline .circleci/config.yml]: Are you sure about this? According to `scan` documentation the default output directory is `./fastlane/test_output`. I see that `circle-ci` for `libextobjc` for example has the same path, but if I look at the output of a [job](https://circleci.com/gh/Lightricks/libextobjc/2) I see: 
```
Unable to save test results from /Users/distiller/project/test_output/report.xml
Error stat /Users/distiller/project/test_output/report.xml: no such file or directory
Found no path with test results, skipping
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      - store_artifacts:
+          path: /tmp/test-results
```
Comment (DekelAvrahami) [inline .circleci/config.yml]: Are you sure about this? I don't see any documentation stating that the tests results are saved to this path. Again I see that `circle-ci` for `libextobjc` has the same path, but again if I look at the output of a [job](https://circleci.com/gh/Lightricks/libextobjc/2) I see:
```
Uploading /tmp/test-results to scan-test-results
  No artifact files found at /tmp/test-results
```

**Review by DekelAvrahami — COMMENTED** [review]
Everything looks OK except couple of issues I found in the `config.yml` file. I understand it is probably copied from another project, but I suspect it is wrong.

**Comment by StatusReport** [comment]
Done.

**Pull request merged**